invenio_sequencegenerator/models.py

```diff
@@ -27,15 +27,15 @@
 from sqlalchemy.orm.exc import MultipleResultsFound
 
 from .errors import InvalidResetCall, InvalidTemplate
-from .utils import double_counter, extract_placeholders
+from .utils import double_counter, extract_placeholders, undouble_counter
 
 
 class TemplateDefinition(db.Model, object):
     """Representation of a template definition."""
```

Comment:
@jirikuncar  Yes, I understand but if I understood correctly  from : 

> +.. automodule:: invenio_sequencegenerator.models
+   :members:

The documentation will generate it from the comments, I really find this docstring not helpful to understand the code :/